src/e2e/java/teammates/e2e/cases/AdminHomePageE2ETest.java

```diff
@@ -47,7 +47,6 @@ public void testAll() {
         assertTrue(failureMessage.contains(
                 "\"invalidemail\" is not acceptable to TEAMMATES as a/an email because it is not in the correct format."));
 
-        assertNotNull(BACKDOOR.getAccountRequest(email, institute));
```

Comment:
unfortunately, there is no way for us to get the UUID of the account request created by the page, so we cannot do this assertion anymore

Reply:
There is a endpoint for all pending requests, we can try to get all pending requests and then filter out this particular request by comparing all the attributes?

Should not be too bad for the purposes of the E2E test since there should not be many account requests in the db?